src/index.html

```diff
@@ -8,6 +8,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <nav class="nav">
+      <ul class="stars stars--0">
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+      </ul>
+
+      <ul class="stars stars--1">
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+
+        <li class="stars__star"></li>
+      </ul>
```

Comment:
```suggestion
      <ul class="stars stars--0">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>

      <ul class="stars stars--1">
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
        <li class="stars__star"></li>
      </ul>
```
You can remove empty lines between li's to make your code easier to read